modules/administration-guide/partials/proc_enabling-readiness-init-containers-for-the-operator-installer.adoc

```diff
@@ -0,0 +1,25 @@
+[id="enabling-readiness-init-containers-for-the-operator-installer_{context}"]
+= Enabling the readiness init containers for the Operator installer
+
+The readiness init containers are not enabled by default, so to use them you first have to enable them. To enable the readiness init containers for {prod-short} installed by the Operator installer:
+
+.Prerequisites
+
+* {prod} installed by the Operator installer.
+
+.Procedure
+
+. Find {prod-short} Operator Deployment name.
```

Comment:
```suggestion
. Find the name of the {prod-short} Operator Deployment.
```

Reply:
@mmorhun It is possible to be a bit more specific here re how to find it (maybe just expand the same sentence or add one more sentence)? Maybe a specific command can get the user this name?

Reply:
Usually, it is `che-operator`, but in some very special cases could be different

Reply:
```suggestion
Find the name of the {prod-short} Operator Deployment. For example, `che-operator`.
```